### DISCUSSION TITLE
Left to Right Scroll

### DISCUSSION THREAD

**Discussion by bryanrsmith36** [KinesisCorporation/Adv360-Pro-ZMK#444] [General]
Has anybody been able to create a macro where I can alter the direction my mouse scrolls by simply pressing and holding a programmed button on my keyboard?

**Comment by huber-th** [comment]
Do you mean direction as in reverse, change up down to down up or change direction as in scroll horizontal vs vertical?

If it's the latter, at least on macOS, holding shift while using the mouse wheel enables horizontal scroll. Something similar might exist in other operating systems.
